src/components/navbar/NavBarStyles.js

```diff
@@ -0,0 +1,79 @@
+import styled from 'styled-components';
+
+export const BtnContet = styled.div`
+	border: 1px solid green;
+	display: inline-block;
+
+	> .bar {
+		width: 30px;
+		margin: 3px;
+		height: 4px;
+		color: rgb(139, 139, 137);
+		background: grey;
+	}
+
+    @media screen and (min-width: 780px) {
+		display: none;
+	}
+`;
+
+export const Nav = styled.div`
+max-width: 1200px;
+margin: 0 auto;
+
+@media screen and (min-width: 780px) {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+`;
+
+export const LogoAndBtn = styled.div`
+display: flex;
+justify-content: space-between;
+align-items: center;
+cursor: pointer;
+`;
+
+export const Logo = styled.div`
+cursor: pointer;
+padding: 15px;
+
+> img {
+    width: 90px;
+}
+`;
+
+export const Navigation = styled.ul`
+height: 0;
+transition: all 0.5s ease;
+overflow: ${(props) => (props.isOpen ? 'visible' : 'hidden')};
+
+
+> li a {
+    background-color: rgb(234, 234, 234);
+    display: block;
+    padding-bottom: 20px;
+    
+    cursor: pointer;
+}
+
+> li a:hover {
+    background-color: #efece1;
+}
+
+@media screen and (min-width: 780px) {
+    overflow: ${(props) => (props.isOpen ? 'visible' : 'visible')};
```

Comment:
no need to do ternary operator here, if both will return `visible`
just say
`overflow: 'visible'`

Reply:
Yes, that's what I thought too.
But it doesn't work, because it is overwritten by the onClick on top, the one it is into the mobile view.
I tried more than one time and in the end I came up with that solution. Actually that is the thing that I wanted to highlight to you.

Reply:
ok, forget it :) 
You were right of course and I don't know what happen yesterday.
I fixed and it works now.